### PR TITLE
{devel}[system/system] ReFrame v3.9.2

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/d16dfd193ac01abe6fb2a9e0d5bf2dc1 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/a04a60c8ab2e9886155c7c287d7e887d for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/fdf30e84ad00c3d04179216c6a07711d for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc01.int.jusuf.sebastian.cluster - Linux rocky linux 8.4, x86_64, AMD EPYC 7742 64-Core Processor (zen2), 1 x NVIDIA GRID V100-4C, 460.73.01, Python 3.6.8
See https://gist.github.com/068a363fd0be6996652cf59ff4555897 for a full test report.

File: easybuild/easyconfigs/r/ReFrame/ReFrame-3.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    }),
+    ('wheel', '0.37.0', {
+        'checksums': ['e2ef7239991699e3355d54f8e968a21bb940a1dbf34a4d226741e64462516fad'],
+    }),
+    ('setuptools', '59.4.0', {
+        'source_tmpl': 'setuptools-%(version)s.tar.gz',
+        'checksums': ['b4c634615a0cf5b02cf83c7bedffc8da0ca439f00e79452699454da6fbd4153d'],
+    }),
+    ('reframe', version, {
+        'preinstallopts': "export PATH=%(installdir)s/bin:$PATH && "
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/r/ReFrame/ReFrame-3.9.2.eb:31]: @boegel wouldn't it be better to keep the comment that was in `modextrapaths`, explaining why this is necessary?

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c201949b69d3cec0f8a5e1fb07450059 for a full test report.

**Comment by branfosj** [comment]
My failure looks similar to the one I see in https://github.com/easybuilders/easybuild-easyconfigs/pull/13844#discussion_r697850516 just with `glob` this time instead of `zipfile`.

In my EB python venv (which is the python being used by the `reframe` script):
```
$ python -c "import glob; print(glob.__file__)"
/usr/lib64/python3.6/glob.py
```

File: easybuild/easyconfigs/r/ReFrame/ReFrame-3.9.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/r/ReFrame/ReFrame-3.9.2.eb:32]: fixed!

**Comment by verdurin** [comment]
Test report by @verdurin
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 33, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.9.9
See https://gist.github.com/25ff54a79f2a0f5081f7dfcca955a406 for a full test report.

**Comment by verdurin** [comment]
```
The following Python packages were likely not installed correctly because they show a version of '0.0.0':
chrome-gnome-shell
This may be solved by using a *-none-any.whl file as the source instead. See e.g. the SOURCE*_WHL templates.
Otherwise you could check if the package provides a version at all (took 2 mins 5 secs)
```

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/8f039278c37d20fda87a60f915fd83aa for a full test report.

**Comment by branfosj** [comment]
I get a [different error now](https://github.com/easybuilders/easybuild-easyconfigs/pull/14477#issuecomment-1155203198):
```
== 2022-06-14 14:41:09,807 easyblock.py:4079 WARNING build failed (first 300 chars): `pip check` failed:
flake8 4.0.1 has requirement importlib-metadata<4.3; python_version < "3.8", but you have importlib-metadata 4.8.3.
```

`pip3 check` on that system:
```
$ pip3 check
No broken requirements found.
```

`pip3 check` inside my EB virtual environment:
```
$ pip3 check
No broken requirements found.
```

**Comment by boegel** [comment]
Closing this, superseded by #15698...

**Pull request closed without merging**